addon/utils/element/estimate-element-height.js

```diff
@@ -1,8 +1,8 @@
-import { assert } from 'smoke-and-mirrors/-debug/helpers';
+import Ember from 'ember';
 
 export default function estimateElementHeight(element, fallbackHeight) {
-  assert(`You called estimateElement height without a fallbackHeight`, fallbackHeight);
-  assert(`You called estimateElementHeight without an element`, element);
```

Comment:
Those files get [removed in test and prod mode](https://github.com/adamjmcgrath/smoke-and-mirrors/blob/develop/index.js#L30), so that import will fail